src/unix/apple/macos/component/arm.rs

```diff
@@ -50,7 +50,11 @@ impl ComponentsInner {
         &mut self.components
     }
 
+    #[allow(unreachable_code)]
     pub(crate) fn refresh_list(&mut self) {
+        // See issue https://github.com/GuillaumeGomez/sysinfo/issues/1279
+        return;
+
```

Comment:
Please comment the content of the function instead.

Reply:
I have now changed it to be commented out, this however triggers many clippy issues. If you're worried about performance when not commenting out the function, don't be, as the function automatically gets optimised out if there is an early return, even at optimisation level 0. Should I try comment out the stuff that's causing clippy issues and use some `#[allow(...)]`s or roll back to an early return?

Reply:
I'm not worried about the performance impact, just found it not great. But I didn't expect this many warnings... Having the `return` would be better indeed.